docs/usage/self-hosted-configuration.md

```diff
@@ -548,6 +548,25 @@ Information provided mainly in debug log level.
 Default execution timeout in minutes for child processes Renovate creates.
 If this option is not set, Renovate will fallback to 15 minutes.
 
+## experimentalFlags
+
+The `experimentalFlags` configuration option streamlines managing experimental settings within Renovate.
+This option replaces the use of environment variables (prefixed with `RENOVATE_X_`), providing a more structured and manageable approach.
+
+Old Approach: Previously, users had to set experimental environment variables like `RENOVATE_X_DOCKER_MAX_PAGES`, and others.
+
+New Approach: With `experimentalFlags`, users can specify experimental settings directly within the configuration file, enhancing clarity and ease of maintenance.
```

Comment:
No need for this - the v38 release notes can document this, and then it's done. We don't want to include this in docs for any longer